src/vs/workbench/contrib/terminal/browser/media/shellIntegration.fish

```diff
@@ -75,7 +75,7 @@ end
 # Sent whenever a new fish prompt is about to be displayed.
 # Updates the current working directory.
 function __vsc_update_cwd --on-event fish_prompt
-	__vsc_esc P "Cwd=$(__vsc_escape_value "$PWD")"
+	__vsc_esc P "Cwd=(__vsc_escape_value "$PWD")"
```

Comment:
This will no longer run `__vsc_escape_value` since `()` in `"` is not a command substitution.

Reply:
Ah, yes, you're right. Changed to backtick.